scripts/get-transactions.js

```diff
@@ -1,285 +1,376 @@
+
+  // Set last confirmed block as toBlock
+
+  if (toBlock == 'latest')
+    toBlock = (await web3.eth.getBlock('latest')).number;
+
+  let fromBlock = DXdaoSnapshot.toBlock + 1;
+
+  if (reset){
+    fromBlock = 7850000;
+    DXdaoSnapshot.fromBlock = fromBlock;
+  }
+
+  DXdaoSnapshot.toBlock = toBlock;
+
+  console.log('Getting from block', fromBlock, 'to block', toBlock);
+
+  async function makeSynchronousRequest(url) {
+    let protocol = http;
+    if(url.substr(0,5)==='https') protocol = https
+  	try {
+  		let http_promise = new Promise((resolve, reject) => {
+    		protocol.get(url, (response) => {
+    			let chunks_of_data = [];
+
+    			response.on('data', (fragments) => {
+    				chunks_of_data.push(fragments);
+    			});
+
+    			response.on('end', () => {
+    				let response_body = Buffer.concat(chunks_of_data);
+    				resolve(response_body.toString());
+    			});
+
+    			response.on('error', (error) => {
+    				reject(error);
+    			});
+    		});
+    	});
+  		return JSON.parse(await http_promise).result;
+  	}
+  	catch(error) {
+  		console.error(error);
+  	}
+  }
+
+  async function asyncForEach(array, callback) {
+    for (let index = 0; index < array.length; index++) {
+      await callback(array[index], index, array);
+    }
+  }
+
+  function getTransactionHashesComposer(
+    _txsUriBuilder,
+    _internalTxsUriBuilder,
+    _getHashFromApiServiceTx,
+    _apiServiceName //for logging only
+  ){
+    return async (_address, _fromBlock, _toBlock) => {
+      let txsFromApiService = await makeSynchronousRequest(
+        _txsUriBuilder(_address, _fromBlock, _toBlock)
+      )
+      let internalTxsFromApiService = await makeSynchronousRequest(
+        _internalTxsUriBuilder(_address, _fromBlock, _toBlock)
+      )
+      let txHashes = txsFromApiService.map(_getHashFromApiServiceTx)
+      let internalTxHashes = internalTxsFromApiService.map(_getHashFromApiServiceTx)
+
+      console.log('Address:',_address,'; Service:',_apiServiceName,'; Tx Hashes:',txHashes.length,'; Int Tx Hashes:', internalTxHashes.length);
+      return { txHashes, internalTxHashes };
+    }
+  }
+
+  function etherscanGetHashFromApiServiceTx(tx){
+    return tx.hash
+  }
+
+  function etherscanTxsUriBuilder(_address, _fromBlock, _toBlock){
+    return 'https://api.etherscan.io/api?module=account&action=txlist&address='
+            +_address
+            +'&startblock='+_fromBlock
+            +'&endblock='+_toBlock
+            +'&sort=asc&apikey='
+            +EtherscanAPIToken
+  }
+
+  function etherscanInternalTxsUriBuilder(_address, _fromBlock, _toBlock){
+    return 'https://api.etherscan.io/api?module=account&action=txlistinternal&address='
+            +_address
+            +'&startblock='+_fromBlock
+            +'&endblock='+_toBlock
+            +'&sort=asc&apikey='
+            +EtherscanAPIToken
+  }
+
+  function blockscoutGetHashFromApiServiceTx(tx){
+    if(typeof(tx.hash)!=='undefined') return tx.hash

```

Comment:
Missing space between teh if and ( and return goes after the break line, no need to do one lines ifs :)